### ISSUE TITLE
Taur colors don't appear properly on station.

### ISSUE THREAD

**Issue by Innocent-Fire** [Skyrat-SS13/Skyrat-TG#16010]
Reporting client version: 514.1583

## Round ID:
6401

## Testmerges:
- [[MODULAR] Fixes That Stupid Language Reset Bug That's Been Here Since Newprefs](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15802)
- [[Tram] New tram, new consequences. Tram collisions actually hurt.](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15842)
- [the ash farming expands](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15898)
- [Fixes events happening too often and blobs being handled outside of dynamic](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15947)
- [Fixes being able to retract an active MODsuit's parts and still be able to use its modules](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15950)

## Reproduction:
Basically when you make a canine, feline, deer, drider, etc taur, it shows the colors as displayed in the picture, but as soon as you wear clothing, it just shows the first mutant color you picked. The secondary color vanishes entirely. Even on station you're stuck to being one solid color. It's not a huge deal, but it's irritating. I'm not entirely sure what broke it, but it could possibly be tied to the new taur limbs or something. Also for some reason being 'naked' also doesn't show the colors sometimes, but being 'aroused' does show the colors. Very strange.

![picture](https://user-images.githubusercontent.com/104606051/188049514-5902c6b8-9a92-45a0-9333-4bd9401d88a0.png)

**Comment by Resreep** [comment]
please fix this im begging you it's been bugged since the refactaur change

**Comment by desertmote** [comment]
fixed by https://github.com/Skyrat-SS13/Skyrat-tg/pull/16871

**Comment by vinylspiders** [comment]
@Jolly-66 fixed see above
